source/options-storage.ts

```diff
@@ -41,7 +41,24 @@ export const renamedFeatures = new Map<string, string>([
 	['quick-pr-diff-options', 'one-click-diff-options'],
 	['quick-review-buttons', 'one-click-review-submission'],
 	['wait-for-build', 'wait-for-checks'],
-	['pull-request-hotkey', 'pull-request-hotkeys'],
+	['pull-request-hotkey', 'pull-request-shortcuts'],
```

Comment:
Let's make sure we merge this PR before the next release too, since https://github.com/refined-github/refined-github/pull/5473 made this rename

Reply:
Rather… let's undo that rename too?